zulipterminal/ui_tools/views.py

```diff
@@ -1330,6 +1330,19 @@ def __init__(self, controller: Any, stream_id: int) -> None:
         ]
 
         total_members = len(stream["subscribers"])
+        Stream_id = stream["stream_id"]
+        type_of_stream = "Private" if stream["invite_only"] is True else "Public"
+        stream_posting_policy = {
+            1: "Any user can post",
+            2: "Only administrators can post",
+            3: "Only full members can post",
+            4: "Only moderators can post",
+        }
+        posting_policy = stream_posting_policy[stream["stream_post_policy"]]
```

Comment:
As per API docs, the `stream_post_policy` field was added in Server version 3.0; there's no feature level mentioned there, so you can either query that in the #api-design stream in chat.zulip.org or you could just check if the field is absent so use the older `is_announcement_only` field.